src/pages/beta/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx

```diff
@@ -122,6 +123,8 @@ const LessonOverviewPage: NextPage<LessonOverviewPageProps> = ({
     expired,
   } = curriculumData;
 
+  const isWorksheetPortrait = !isWorksheetLandscape;
```

Comment:
Could you change the prop in OverviewPresentation rather than creating another variable here?